data/json/recipes/recipe_weapon.json

```diff
@@ -6733,5 +6733,52 @@
     ["steel_chunk", 4],
     ["scrap", 12]]
   ]
+},
+{
+  "type" : "recipe",
+  "result": "chemical_thrower",
+  "category": "CC_WEAPON",
+  "subcategory": "CSC_WEAPON_RANGED",
+  "skill_used": "mechanics",
+  "skills_required": [[ "gun", 3 ], [ "cooking", 3]],
+  "difficulty": 7,
```

Comment:
That's quite a lot of requirements there.
I'd drop the gun requirement. Maybe change it to `[[ "launchers", 1 ]]` or 2.
And 5 mechanics would also be plenty - 7 is for complex, precise stuff. Since the thrower is really bulky, you could also make it more "makeshift".

Early availability is quite important for this kind of stuff, because later on it will be obsoleted by the chain lightning CBM.